runtime/vm/classallocation.c

```diff
@@ -145,6 +152,20 @@ internalAllocateClassLoader(J9JavaVM *javaVM, j9object_t classLoaderObject)
 			classLoader->flags |= J9CLASSLOADER_PARALLEL_CAPABLE;
 		}
 	}
+
+#if JAVA_SPEC_VERSION >= 9
+	if (NULL != classLoaderClass) {
+		J9UTF8 *classLoaderClassNameUTF = J9ROMCLASS_CLASSNAME(classLoaderClass->romClass);
```

Comment:
Please avoid checking the class name for every classloader created. There is only one Platform classloader, once it is found the VM can stop checking.

Reply:
Note the VM already stores the platform classloader in the J9JavaVM.extensionClassLoader field.

Reply:
Like systemClassLoader, should i cache platformClassLoader in J9JavaVM?

Reply:
added new changes: use J9JavaVM.extensionClassLoader to ignore PlatformClassLoader.